swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+'''
+Clothing class is a child of the Item class, imported above. 
+Differences from parent include unique __str__ method and category attribute hard-coded to "Clothing".
+'''
+
+
+class Clothing(Item):
+    def __init__(self, condition=0, age=None):
+        super().__init__(category="Clothing", condition=condition, age=age)
```

Comment:
👍  Good work in ensuring that all `Clothing` instances have a `category` of `"Clothing"` by passing the value into the parent constructor.